LiipImagineBundle.php

```diff
@@ -45,6 +46,7 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new PostProcessorsCompilerPass());
         $container->addCompilerPass(new ResolversCompilerPass());
         $container->addCompilerPass(new MetadataReaderCompilerPass());
+        $container->addCompilerPass(new NonFunctionalFilterExceptionPass());
```

Comment:
As i mentioned in the comment of this MR, I think this pass should be added first. Otherwise an error occurs:

```
The service "liip_imagine.filter.manager" has a dependency on a non-existent service "liip_imagine.filter.loader.paste". 
```

Reply:
Thanks for this review! I have now moved the compiler pass to the top.